### PR TITLE
PRS 23 24

### PR DESCRIPTION
**Pull request by DenisNosnitsin1970**
New entries

### CODE REVIEW

File: new/PRS13123TaklaEwostatewos.xml [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+    </teiHeader>
+    <text>
+        <body>
+            <listPerson>
+                <person sex="1">
+                    <persName xml:lang="gez" xml:id="n1">ተክለ፡ ኤዎስጣቴዎስ፡</persName>
+                    <persName xml:lang="gez" type="normalized" corresp="#n1">Takla ʾEwosṭātewos</persName>
+                    <persName xml:lang="gez" xml:id="n2">ተክለ፡ ኤዎስጣቴዎስ፡ ወልደ፡ ጐቢስ፡</persName>
+                    <persName xml:lang="gez" type="normalized" corresp="#n2">Takla ʾEwosṭātewos walda Gʷabis</persName>
+                    <persName xml:lang="en" type="normalized" corresp="#n2">Takla ʾEwosṭātewos son of Gʷabis</persName>
```
Comment (eu-genia) [inline new/PRS13123TaklaEwostatewos.xml:52]: take out type="normalized" from the en version

File: new/PRS13123TaklaEwostatewos.xml [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+                    <persName><roleName type="title">sǝyyuma Dabra Māʿḍo</roleName>Takla ʾEwosṭātewos</persName>
```
Comment (eu-genia) [inline new/PRS13123TaklaEwostatewos.xml:53]: space before Takla

Comment (eu-genia) [inline new/PRS13123TaklaEwostatewos.xml:53]: no negusa s?

File: new/PRS13123TaklaEwostatewos.xml [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+                    <occupation type="ruler"/>
+                    <nationality type="Ethiopia"/>
+                    <birth/>
+                    <death/>
+                    <floruit notBefore="1600" notAfter="1670"/>
+                </person>
+                <listRelation>
+                    <relation name="saws:hasOwned" active="PRS13123TaklaEwostatewos" passive="ESmy18"/>
+                    <relation name="saws:isRelatedTo" active="PRS13123TaklaEwostatewos" passive="LOC2320DabraM"/>
+                    </listRelation>
```
Comment (eu-genia) [inline new/PRS13123TaklaEwostatewos.xml:64]: no entry for the father?

File: new/PRS13124GabraKrestos.xml [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+        <revisionDesc>
+            <change who="DN" when="2020-08-04">Created entity</change>
+        </revisionDesc>
+    </teiHeader>
+    <text>
+        <body>
+            <listPerson>
+                <person sex="1">
+                    <persName xml:lang="gez" xml:id="n1">ገብረ፡ ክርስቶስ፡</persName>
+                    <persName xml:lang="gez" type="normalized" corresp="#n1"> Gabra Krǝstos</persName>
```
Comment (eu-genia) [inline new/PRS13124GabraKrestos.xml:49]: delete space before Gabra

File: new/PRS13123TaklaEwostatewos.xml (hunk 1 above)
Comment (DenisNosnitsin1970) [reply new/PRS13123TaklaEwostatewos.xml:52]: ok

File: new/PRS13123TaklaEwostatewos.xml (hunk 2 above)
Comment (DenisNosnitsin1970) [reply new/PRS13123TaklaEwostatewos.xml:53]: ok

Comment (DenisNosnitsin1970) [reply new/PRS13123TaklaEwostatewos.xml:53]: no, it was not in the original text

File: new/PRS13123TaklaEwostatewos.xml (hunk 3 above)
Comment (DenisNosnitsin1970) [reply new/PRS13123TaklaEwostatewos.xml:64]: I do not know, I would not do it, unless he appears elsewhere.

File: new/PRS13124GabraKrestos.xml (hunk 4 above)
Comment (DenisNosnitsin1970) [reply new/PRS13124GabraKrestos.xml:49]: ok

**Pull request merged**